packages/@aws-cdk/cloudformation-include/test/nested-stacks.test.ts

```diff
@@ -0,0 +1,528 @@
+import { ResourcePart } from '@aws-cdk/assert';
+import '@aws-cdk/assert/jest';
+import * as s3 from '@aws-cdk/aws-s3';
+import * as core from '@aws-cdk/core';
+import * as path from 'path';
+import * as inc from '../lib';
+import * as futils from '../lib/file-utils';
+
+// tslint:disable:object-literal-key-quotes
+/* eslint-disable quotes */
+
+describe('CDK Include', () => {
+  let stack: core.Stack;
+
+  beforeEach(() => {
+    stack = new core.Stack();
+  });
+
+  test('can ingest a template with one child', () => {
+    const parentTemplate = new inc.CfnInclude(stack, 'ParentStack', {
+      templateFile: testTemplateFilePath('parent-onechild.json'),
+      nestedStacks: {
+        'ChildStack': {
+          templateFile: testTemplateFilePath('grandchild-import-stack.json'),
+        },
+      },
+    });
+
+    const childStack = parentTemplate.getNestedStack('ChildStack');
+    expect(childStack.stack).toMatchTemplate(
+      loadTestFileToJsObject('grandchild-stack.expected.json'),
```

Comment:
This file is exactly the same as `grandchild-import-stack.json`. What is going on here? If they should be the same, you should use the same file in the assertion as you do for the input...